cg/apps/demultiplex/sample_sheet/models.py

```diff
@@ -21,7 +21,7 @@ class FlowCellSample(BaseModel):
     sample_id: SampleId
     index: str
     index2: str = ""
-    model_config = ConfigDict(populate_by_name=True, extra=Extra.ignore)
+    model_config = ConfigDict(populate_by_name=True, extra="ignore")
```

Comment:
Would we want to create a constant for "ignore?

Reply:
Maybe if it is used elsewhere, but it is not now so I think it is fine